spec/services/search_identities_spec.rb

```diff
@@ -206,8 +206,8 @@
     end
 
     context 'cro_number search' do
-      context 'when query matches' do
```

Comment:
If you do not pass wildcards, the search will be performed the same way as before